### PR TITLE
Update allowfullScreen, youtube link in watchSecondary

### PR DESCRIPTION
**Pull request by HolmesJJ**

### CODE REVIEW

File: my-website/docs/cloneCozie.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ In order to edit the source code and being able to work on Cozie you will need t
 
 Please follow the instructions in the video below.
 
-<iframe width="100%" height="315" src="https://www.youtube.com/embed/gSNPvoGc8Zw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (FedericoTartarini) [inline my-website/docs/cloneCozie.md:13]: Are you sure about this code change? When I try to share an iframe from YouTube, `allowfullsreen` is all lowercase. See image below.

image

**Review by FedericoTartarini — COMMENTED** [review]
Please review my comment since from YouTube it appears that `allowfullscreen` should be lowercase. Thank you for changing the link to the video.

**Comment by FedericoTartarini** [comment]
@HolmesJJ could you please review my comments so I can accept your pull request?

**Comment by HolmesJJ** [comment]
I cannot click the fullscreen icon as shown below, and the allowfullscreen attribute disappeared as well:
a1bea4eda61e39938822e8ecf4f7a18

When I update S and F to uppercase and run the website at localhost, it works, but it still shows all lowercase at the chrome browser:
image
image

I think this is the reason for it:
https://stackoverflow.com/questions/37340052/youtube-full-screen-is-unavailable

**Review by FedericoTartarini — APPROVED** [review]

**Pull request merged**